docs/source/specs/openapi.json

```diff
@@ -110,6 +110,16 @@
           "401": {
             "description": "Unauthorized"
           },
+          "403": {
+            "description": "Insufficent permissions to list principals",
```

Comment:
Looks like the spelling of "Insufficent" to "Insufficient" could be updated throughout.